### PR TITLE
Use Point, Rect and Size member functions instead of global functions

### PR DESCRIPTION
**Pull request by minggo**

### CODE REVIEW

File: tools/tolua++/CCUserDefault.pkg [hunk 1]
```diff
@@ -17,12 +17,7 @@ class CCUserDefault
 
 	void flush();
 
-    // XXX: deprecated
     static CCUserDefault* sharedUserDefault();
     static void purgeSharedUserDefault();
-
-    static CCUserDefault* getInstance();
-    static void destroyInstance();
```
Comment (dumganhar) [inline tools/tolua++/CCUserDefault.pkg:25]: ？delete?

File: tools/tolua++/CCTextureCache.pkg [hunk 2]
```diff
@@ -13,12 +13,7 @@ class CCTextureCache : public CCObject
     void removeTextureForKey(const char *textureKeyName);
     void dumpCachedTextureInfo();
 
-    // XXX: deprecated
     static CCTextureCache * sharedTextureCache();
-    static void purgeSharedTextureCache();
-
-    static CCTextureCache * getInstance();
-    static void destroyInstance();
```
Comment (dumganhar) [inline tools/tolua++/CCTextureCache.pkg:21]: Why delete?

File: tools/tolua++/CCSpriteFrameCache.pkg [hunk 3]
```diff
@@ -16,10 +16,6 @@ class CCSpriteFrameCache : public CCObject
 
     CCSpriteFrame* spriteFrameByName(const char *pszName);
 
-    // XXX deprecated
-    static CCSpriteFrameCache* sharedSpriteFrameCache();
-    static void purgeSharedSpriteFrameCache();
-
-    static CCSpriteFrameCache* getInstance();
-    static void destroyInstance();
```
Comment (dumganhar) [inline tools/tolua++/CCSpriteFrameCache.pkg:24]: I think you resolve conflicts with wrong side.

File: tools/tolua++/CCFileUtils.pkg [hunk 4]
```diff
@@ -1,14 +1,9 @@
 
 class CCFileUtils
 {
-    /** @deprecated Use getInstance() instead */
     static CCFileUtils* sharedFileUtils();
-    /** @deprecated Use destroyInstance() instead */
     static void purgeFileUtils();
     
-    static CCFileUtils* getInstance();
-    static void destroyInstance();
```
Comment (dumganhar) [inline tools/tolua++/CCFileUtils.pkg:10]: I think you resolved conflicts with wrong side.

File: tools/tolua++/CCEGLViewProtocol.pkg [hunk 5]
```diff
@@ -86,8 +86,6 @@ class CCEGLViewProtocol
 
 class CCEGLView : public CCEGLViewProtocol
 {
-    /** @deprecated Use getInstance() instead */
-    static CCEGLView* sharedOpenGLView();
-    static CCEGLView* getInstance();
+    static CCEGLView* sharedOpenGLView(void);
```
Comment (dumganhar) [inline tools/tolua++/CCEGLViewProtocol.pkg:90]: I think you resolve conflicts with wrong side.

File: tools/tolua++/CCDirector.pkg [hunk 6]
```diff
@@ -57,7 +57,5 @@ class CCDirector : public CCObject
 	CCSize getVisibleSize();
     CCPoint getVisibleOrigin();
 
-    /** @deprecated Use getInstance() instead */
-    static CCDirector* sharedDirector();
-    static CCDirector* getInstance();
+    static CCDirector* sharedDirector(void);
```
Comment (dumganhar) [inline tools/tolua++/CCDirector.pkg:61]: boom.

File: tools/tolua++/CCAnimationCache.pkg [hunk 7]
```diff
@@ -5,13 +5,8 @@ class CCAnimationCache : public CCObject
...
 
-    // XXX deprecated, use getInstance instead
-    static CCAnimationCache* sharedAnimationCache();
-    // XXX deprecated, use destroyInstance instead
-    static void purgeSharedAnimationCache();
-
-    static CCAnimationCache* getInstance();
-    static void destroyInstance();
+    static CCAnimationCache* sharedAnimationCache(void);
+    static void purgeSharedAnimationCache(void);
```
Comment (dumganhar) [inline tools/tolua++/CCAnimationCache.pkg:11]: boom

**Comment by minggo** [comment]
Yep, it is strange.
I didn't modify these files.

I will fix it soon.

**Pull request merged**